deploy/start.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+export MIGRATION_PLANNER_API="${MIGRATION_PLANNER_API:-'http://migration-planner:3443'}"
```

Comment:
IIRC, this will require running the container with the --add-host flag, `podman run --add-host migration-planner:<some-ip> <this-image-name>`. Prhaps it could be documented somewhere...

Reply:
Yeah, I was testing it with: ```podman run -e MIGRATION_PLANNER_API=http://127.0.0.1:3443``` We need to document it.